src/nav/ChatNavBar.js

```diff
@@ -31,19 +32,46 @@ class ChatNavBar extends PureComponent<Props> {
       backgroundColor === DEFAULT_TITLE_BACKGROUND_COLOR
         ? BRAND_COLOR
         : foregroundColorFromBackground(backgroundColor);
+    const spinnerColor =
+      backgroundColor === DEFAULT_TITLE_BACKGROUND_COLOR
+        ? 'default'
+        : foregroundColorFromBackground(backgroundColor);
 
     return (
-      <View style={[styles.navBar, { backgroundColor }]}>
-        <NavButton
-          name="arrow-left"
-          color={color}
-          onPress={() => {
-            dispatch(navigateBack());
-          }}
+      <View
+        /* eslint-disable react-native/no-inline-styles */
+        style={{
+          borderColor: 'hsla(0, 0%, 50%, 0.25)',
+          borderBottomWidth: 1,
+        }}
+      >
+        <View
+          /* eslint-disable react-native/no-inline-styles */
+          style={[
+            {
+              flexDirection: 'row',
+              height: NAVBAR_SIZE,
+              alignItems: 'center',
+            },
+            { backgroundColor },
+          ]}
+        >
```

Comment:
How about making this split in its own little NFC commit, and then adding the `LoadingBanner` after that? I think that'd make it easier to read, separately, the part that's just pulling these two pieces apart from each other and the part that's inserting the `LoadingBanner` (and the `spinnerColor` logic for it.)